embark/lib/templates/README.md.eex

```diff
@@ -1,41 +1,64 @@
 # <%= app_module %>
 
-### Start services
+## Set Environment Variables
+
+Open the `hello_tokumei/.env` file to change environment variables to match your needs.
+
+
+### Https Port
+
+To override the default port from `8443` to `9443`:
+
+```
+HTTPS_PORT=9443
```

Comment:
is HTTPS_PORT a more standard convention than SECURE_PORT. makes sense to use just one throughout?

Reply:
Used that name because is more explicit, but I can change it to match your one.